src/main/java/com/neighborhoodAssurance/dto/NameDTO.java

```diff
@@ -2,4 +2,61 @@
 
 public class NameDTO {
 

```

Comment:
I kind of assume here that you'll be receiving an array list of neighborhood json data based on crimeRate / year pairs.
Here's an example of what I was envisioning:
[{ id: 
name: 
city:
state:
crimeRate:
year:
}]

So then you could sort similar data by using city, state, or neighborhood.  Then, you could compile the crime rates by year and average / compare them as needed.